조도센서 측정.

HeejuYoon opened the pull request:
조도센서 측정 값에 따라 3색 led등 불켜기

HeejuYoon committed:
조도센서 측정.

조도센서 측정 값에 따라 3색 led등 불켜기

dasg34 commented on light.ino:
```diff
@@ -34,8 +45,60 @@ light_warn()
+    analogWrite(RED, 255);
+  }
+
+  else if(light_warning_count>=3 && light_warning_count<6)
```
뒤에 있는 '&& light_warning_count<6' 조건은 필요 없습니다!

HeejuYoon committed:
조도센서 측정 후 3색led 불켜기

조도센서로 측정값 확인하기.
측정값에 따라 3색 led등 불 켜기.
(어두운게 6번이상 지속 - 빨간색
3번이상 6번미만 - 초록색
3번 미만의 경우 - 파란색)

HeejuYoon committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into develop-light

HeejuYoon committed:
조도센서 측정 값에 따라 3색 led등 불켜기_수정1
